packages/flame/test/game/game_widget/game_widget_keyboard.dart

```diff
@@ -0,0 +1,156 @@
+import 'dart:ui';
```

Comment:
This file is missing the test suffix on its name

Reply:
Still missing

Reply:
when i did this mistake in the past, `flutter test` didn't run it (making me very confused haha)